blocks/core/ff_container/ff_container-dashboard-section/ff_container-dashboard-section.less

```diff
@@ -1,38 +0,0 @@
-//header
```

Comment:
@dkhuntrods I'm not sure that's the case here — I don't even think this file should exist. The only core styles _should_ be things like layout — as this doesn't have layout that's shared between themes, it doesn't need core styling. Unless I'm missing something?